assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
feels like `.trim()` is guarding against developer errors, IMO i'd prefer it to not work and then the developer fixes the not needed whitespace inside the `href`.